p/themes/base-theme/frss.rtl.css

```diff
@@ -2421,10 +2437,8 @@ html.slider-active {
 	}
 
 	.dropdown .dropdown-menu {
-		width: 94%;
-		border-radius: 0;
-		right: 3%;
-		left: 3%;
+		right: 0.5rem !important;
+		left: 0.5rem !important;
```

Comment:
Any possibility of avoiding `!important` here?

Reply:
unfortunatelly not :/
It is needed to overrule the following lines that are more specific

![grafik](https://github.com/user-attachments/assets/84a1578e-5e8b-4a4c-828d-0c10d2f13e60)